autoware_launch/CMakeLists.txt

```
@@ -15,3 +15,26 @@ ament_auto_package(
   launch
   rviz
 )
+
+# Copy the script to the build directory and ensure it is executable
+add_custom_target(copy_qt5ctsetup ALL
```

The following may be better since there is an error in spell check.
```suggestion
add_custom_target(copy_qt5ct_setup ALL
```